src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -105,8 +105,16 @@ public function match(\$pathinfo)
     {
         \$allow = array();
         \$pathinfo = rawurldecode(\$pathinfo);
+        \$trimmedPathinfo = rtrim(\$pathinfo, '/');
         \$context = \$this->context;
         \$request = \$this->request;
+        \$requestMethod = \$isLikeGetMethod = \$context->getMethod();
+        \$schema = \$context->getScheme();
+
+        if ('HEAD' === \$requestMethod) {
+            \$isLikeGetMethod = 'GET';
```

Comment:
What about renaming it to `$methodAlias` or `$canonicalMethod` or something like that? We "can't" store a string in a `$is*` variable.